app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -67,4 +71,6 @@ private fun LessonsHeader() = Column {
         style = MaterialTheme.typography.bodyMedium,
         modifier = Modifier.padding(top = 4.dp)
     )
+    HorizontalDivider(modifier = Modifier.padding(vertical = 12.dp))
+    LessonFilters()
```

Comment:
what I meant by the comment about this was to include in the `item("header", "header") {}` block directly, like this:

```
item("header", "header") {
    LessonsHeader()
    HorizontalDivider(modifier = Modifier.padding(vertical = 12.dp))
    LessonsFilters()
}
```